lib/openscap/xccdf/item.rb

```diff
@@ -64,6 +64,17 @@ def rationale(prefered_lang = nil)
         rationale
       end
 
+      def idents
+        idenss = []
```

Comment:
Did you perhaps mean to say `indents` ? Not sure whether this is a typo or legit.

Reply:
Sure, I can fix this one for you.